src/dlgVideoPlayer.cpp

```diff
@@ -0,0 +1,60 @@
+/***************************************************************************
+ *   Copyright (C) 2022 by Mike Conley - mike.conley@stickmud.com          *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ *   This program is distributed in the hope that it will be useful,       *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU General Public License for more details.                          *
+ *                                                                         *
+ *   You should have received a copy of the GNU General Public License     *
+ *   along with this program; if not, write to the                         *
+ *   Free Software Foundation, Inc.,                                       *
+ *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ ***************************************************************************/
+
+
+#include "dlgVideoPlayer.h"
+
+#include "Host.h"
+#include "TMedia.h"
+
+#include "pre_guard.h"
+#include "ui_videoplayer.h"
+#include "post_guard.h"
+
+dlgVideoPlayer::dlgVideoPlayer(QWidget* parent, Host* pH, TMedia* pM) : QWidget(parent), mpHost(pH), mpMedia(pM)
```

Comment:
For non-constant initialisers that remain for a class (after moving the constant ones to the header file) we (well, ***I***) like to put each one on a single line - as it make it easier to track changes in git - and to view on developers' systems when they do not have humongous wide-screens:
```suggestion
dlgVideoPlayer::dlgVideoPlayer(QWidget* parent, Host* pH, TMedia* pM)
: QWidget(parent)
, mpHost(pH)
, mpMedia(pM)
```